docs/src/man/tutorial_time_Seismicity.md

```diff
@@ -0,0 +1,87 @@
+
+![Tutorial_SeismicTime_Geomorphology](../assets/img/Geomorphology_Velocity.png)
+
+To reproduce it, represent the coastline as data points with black solid color and assign your favourite color map to the morphology. Note that each block color number corresponds to a different morphology. Beware that this file only works in cartesian coordinate, as it is still impossible to generate shape files in real UTM coordinates. To reproduce exactly the figure, you will need the velocity model loaded in the km-scale volcano tutorial.
+
+#### 3. Earthquakes in movie
+
+Create the folder **TemporalSeismicity** in the current folder and open the movie with the function *Movie_Paraview*.
+
+```julia
+julia> using DelimitedFiles, GeophysicalModelGenerator, Dates
+julia> p2                  = @__FILE__;
+julia> p2last              = findlast("/",p2);
+julia> p3                  = chop(p2,head=0,tail = length(p2)-p2last[1]+1);
+julia> output_path         = string(p3,"/");
+julia> movie               = Movie_Paraview(name=string(p3,"/TemporalSeismicity"), Initialize=true);
+julia> if isdir(string(p3,"/TemporalSeismicity"))==0
+julia>     mkdir(string(p3,"/TemporalSeismicity"));
+julia> end
+```
+Now let's load the earthquake data provided as text files. The first column gives us a temporal marker we can use to plot earthquakes in different periods. We used the *Date* package to transform this time into dates.
+
+```julia
+julia> data                = readdlm("SeismicLocations/Seismicity_UTM_1983_1984.txt", '\t', skipstart=0, header=false);
+julia> l                   = length(data[:,1]);
+julia> dates               = Date.(zeros(l,1))
+julia> for i = 1:l
+julia>     df              = DateFormat("yyyymmdd");
+julia>     t1              = string("19",@sprintf("%5.o", data[i,1]));
+julia>     t2              = Date(t1[1:8],df);
+julia>     dates[i,1]      = t2;
+julia> end
+julia> WE                  = data[:,2];
+julia> SN                  = data[:,3];
+julia> depth               = data[:,4];
+julia> dates_num           = dates - dates[1];
+```
+Select earthquakes every 50 days from the starting date (17/01/1983) and use them to create the frames for the video.
+
+```julia
+julia> nt                  = 50;
+julia> dt                  = Dates.Day(nt);
+julia> t                   = minimum(dates):dt:maximum(dates);
+julia> for itime = 1:length(t)-1
+julia>    name            = string(p3,"/TemporalSeismicity/", string(itime));
+julia>    tt=findall(x->(x>=t[itime]) & (x<=t[itime+1]),dates);
+julia>    we             = WE[tt];
+julia>    sn             = SN[tt];
+julia>    Depth1          = depth[tt];
+julia>    DN              = dates_num[tt];
+julia>    label_time      = Dates.value(DN[end]);
+julia>    if size(tt,1)>1
+julia>        Data_set    = CartData(we, sn, Depth1, 33, true, (Depth=Depth1*km,Timedata=DN));
+julia>        movie       = Write_Paraview(Data_set, name,pvd=movie,time=label_time,PointsData=true);
```

Comment:
If I understand it correctly, the data you actually read into `GMG` is in UTM coordinates. If that is the case, please load it into a `UTMData` structure instead You can always convert this to the correct `CartData` structure using a `ProjectionPoint` and `Convert2CartData`.

I recall that your argument to not do this is that the `ShapeFiles` in paraview are only shown in `UTM` coordinates. This can, in fact, be transferred to the correct distances, by shifting that data in Paraview and dividing it by 1000 (using the `Transform` filter.

Things have to be in the correct `CartData` form to generate consistent LaMEM model input.